theme/themes/eea/extras/blockquote.variables

```diff
@@ -16,7 +16,7 @@
 @mobileQuoteFontSize   : 0.688rem;
 @tabletQuoteFontSize   : @h6;
 @computerQuoteFontSize : @h5;
-@quoteFontWeight       : 700; 
+@quoteFontWeight       : 400;
```

Comment:
Rollback your change here,  meta has 400 weight, and the quote itself should be bold while the **meta** section
aka **author** tag is lighter.